components/input.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export function Input ({
+	dataTrackable = 'generic-input',
+	errorText = 'Please enter a value',
+	fieldId = 'genericInputField',
+	hasError = false,
+	inputId = 'genericInput',
```

Comment:
Does it make sense to have default values for dataTrackable, fieldId and inputId?
I don't think we ever want DOM elements with input `id="generidInput"`.
And `data-trackable = 'generic-input'` would bring no value.

So now the question is do we need to allow/handle missing prop?
Like if dataTrackable is null, the element won't have it?
It means we could also allow for elements with no id, if they are not needed.

```suggestion
	dataTrackable,
	errorText = 'Please enter a value',
	fieldId,
	hasError = false,
	inputId,
```